docs/source/index.rst

```diff
@@ -3,7 +3,7 @@ luisy
 
 `luisy` is a Python framework that extends luigi and further
 simplifies building data science pipelines by reducing development
-complexity. Thus, it makes **LUI**gy more ea**SY**.
+complexity. Thus, it makes **LUI**gi more ea**SY**.
```

Comment:
I agree with the change, but that line does still not get formatted correctly in the docu...